src/baldr/graphreader.cc

```diff
@@ -106,6 +140,9 @@ GraphReader::tile_extract_t::tile_extract_t(const boost::property_tree::ptree& p
   }
 }
 
+// TODO: delete this and call the extract constructor directly from within graphreader
+// before we do this we need to figure out what to do with traffic tile extract
+// they should support index reading as well for fast reload
```

Comment:
ah was looking for that comment!

I don't have a traffic tile extract handy, I hardly know their folder layout when they're not archived. you think we could use the current `valhalla_build_extract` script would work as it is right now to produce a new traffic tar? I guess it should, seems like graphreader expects the exact same path/folder layout as for the routing tile extract. the valhalla.json is missing a `mjolnir.traffic_dir` though, to know which directory will be archived.